iRonCub-Mk3/calibrators/left_arm-calib.xml

```diff
@@ -19,11 +19,11 @@
 <param name="calibrationType">        10          10          10         10          5       12      12      7       7       6       6       6       6       6       6       6       </param>
 <!-- <param name="calibration1">           -54795      22399       44679      53215    -1500     -8335   10671    0       0       0       0       0       0       0       0       0       </param> -->
 <param name="calibration1">           4000        -3000       -3000      4000    -1500     63200   64047     0       0       0       0       0       0       0       0       0       </param>
-<param name="calibration2">	       0           0           0          0        16384      0       0        0       0       9102    9102    9102    9102    9102    9102    10000    </param>
+<param name="calibration2">	          0           0           0          0        16384      0       0        0       0       9102    9102    9102    9102    9102    9102    10000    </param>
 <param name="calibration3">           0           0           0          0           0       0       0       0       0      -1        1       -1      1      -1       1       1      </param>
 <param name="calibration4">           0           0           0          0           0       0       0       1970    325     250     478     243     478     255     480     495     </param>
 <param name="calibration5">           0           0           0          0           0       0       0       2090    2730    9       99       15      30       30      90    103     </param>
-<param name="calibrationZero">        25       -8.4        -51.3       -5            0       0      0       0       0       0       0        0        0       0       0       0       </param>
+<param name="calibrationZero">        35          0       -51.3       -5            0       0      0       0       0       0       0        0        0       0       0       0       </param>
```

Comment:
You did not modify the [`right_arm`](https://github.com/ami-iit/robots-configuration/pull/50/files#diff-7f37a6d4ad6df655944c26a7aef6a21a1d62649f5539aeb95408faecaa9d10ac)
I suggest to modify both or no one :)